packages/babel-preset-react-app/create.js

```diff
@@ -82,7 +82,7 @@ module.exports = function (api, opts, env) {
           // Allow importing core-js in entrypoint and use browserlist to select polyfills
           useBuiltIns: 'entry',
           // Set the corejs version we are using to avoid warnings in console
-          corejs: 3,
+          corejs: 3.5,
```

Comment:
```suggestion
          corejs: '3.5',
```

Reply:
Thanks @NMinhNguyen you're correct. Have accepted the suggestions.